feature-reference.md

```diff
@@ -151,5 +152,157 @@ working on a solution that - in extreme circumstances - can be used to import sy
+You can customize the file path with the `-config` CLI flag.
+The file content is expected to be in _plain_ format, as in:
+        
+        flag value1
+        anotherflag value2
+  
+This is the configuration mode used in Kubernetes charts, via a `configMap`.
+
+### Environment variables
+
+The host-agent can be configured with environment variables prefixed with `PRODFILER_`.
+Every CLI flag can be turned into an environment variable that will be parsed with the following conversion:
+
+* remove the flag's first hyphen
+* replace all remaining hyphens with an underscore
+* capitalize all letters
+* prefix the flag with `PRODFILER_`
+
+For example:
+
+| flag | env variable |
+|------|--------------|
+| `-config` | `PRODFILER_CONFIG` |
+| `-project-id` | `PRODFILER_PROJECT_ID`|
+
+### Description of configuration options
+```
+  -cache-directory string
+   The directory where prodfiler can store cached data. (default "/var/cache/optimyze")
+```
+* This folder must exist on the host where host-agent runs, it will not be created autoamtically by the binary but it's
+  part of the OS packages and Kubernetes install process. 
+
+```
+  -collection-agent string
+        The collection agent address in the format of host:port. (default "data.run.prodfiler.com:443")
+```
+* Address of collection-agent: customize _only_ if you know you need to send data in a non-default location
+
+```
+  -config string
+        Path to the prodfiler configuration file. (default "/etc/prodfiler/prodfiler.conf")
+```
+* Path of the configuration file.
+
+```
+  -copyright
+        Show copyright and short license text.
+```
+* Prints the copyright notice and exit
+
+```
+  -disable-tls
+        Disable encryption for data in transit.
+```
+
+* Disables TLS encryption for the communication with collection-agent: set it _only_ if you know you need to send data in
+  an unencrypted, non-default location.
+
+```
+  -no-kernel-version-check
+        Disable checking kernel version for eBPF support. Use at your own risk, to run the agent on older kernels with backported eBPF features.
+```
+
+* Allows experimentally running the host-agent on unsupported kernels, disabling the kernel version check.
+
+```
+  -project-id int
+        The project ID to store the data in.
+```
+
+* The project ID where profiles fetched by the host-agent will be stored; instructions in the UI will fill it for you.
+
+```
+  -secret-token string
+        The secret token associated with the project id.
+```
+
+* The secret token to authenticate with the project ID where profiles will be stored; instructions in the UI will fill
+  it for you.
+
+```
+  -tags string
+        User-specified tags separated by ';'. Each tag should match '^[a-zA-Z0-9-:._]+$'.
+```
+
+* User-provided tags to mark profiles for a given host: these tags can be used to filter data in the UI as mentioned
+  in [filtering](filtering.md#host-derived-keys).
+
+```
+  -t string
+        Shorthand for -tracers. (default "all")
```

Comment:
For the purpose of this documentation I think it can be confusing to have short options for some configuration options, but not for all. So I suggest to remove it here.
```suggestion
```